qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash

```diff
@@ -93,6 +93,12 @@ fi
     chown -R elasticsearch:elasticsearch "$ESPLUGINS"
 
     install_jvm_example
+    start_elasticsearch_service
```

Comment:
I wonder if you can get away with doing this particular check while elasticsearch is started with all the plugins installed? Its fine if we can't though.